sdks/java/fn-execution/src/main/java/org/apache/beam/sdk/fn/stream/DirectStreamObserver.java

```diff
@@ -101,6 +101,11 @@ public void onNext(T value) {
                     + "this issue.",
```

Comment:
"https://issues.apache.org/jira/browse/BEAM-4280 for the history for this issue."

shall we update this link (to a GitHub Issue) and also print in the warn message / or remove the outdated JIRA link at INFO message

Reply:
I don't think it is worthwhile history or filing an issue. I just tried to update comment to summarize what was going on. PTAL